store/store.go

```diff
@@ -378,3 +378,28 @@ func (s *store) WriteBatch() error {
 func (s *store) IsBanned(addr crypto.Address) bool {
 	return s.config.BannedAddrs[addr]
 }
+
+func (s *store) pruneBlock(batch *leveldb.Batch, blockHeight uint32) (bool, error) { //nolint
```

Comment:
You can get batch from `s *store` receiver, just get block height as parameter.

Reply:
It's defined at issue details. wierd....
let me update it.

Reply:
fixed